source/computer/wsl-setup.rst

```diff
@@ -1,18 +1,155 @@
 WSL 配置指南
 ============
 
+:本节贡献者: |赵志远|\（作者）、
+             |姚家园|\（审稿）、
+             |田冬冬|\（审稿）
+:最近更新日期: 2021-01-18
+:预计花费时间: 120 分钟
+
+简介
+-----
+
+WSL 有 WSL1 和 WSL2 两个发行版本，二者底层原理不同。大多数情况下，建议使用 WSL2，
+因为它提供更快的性能和 100% 的系统调用兼容性。但是，当涉及到跨系统的文件互访时
+（Linux 访问 Windows 里的文件，或 Windows 访问 Linux 里的文件），使用 WSL1 具有
+更快的性能。
+
+官方目前没有弃用 WSL1 的计划，并且支持将任何一个已经安装的 Linux 发行版转换为 WSL1 或者 WSL2。
+
+安装 WSL
+---------------
+
+WSL 的安装方式可以参考官方文档：
+
+- 中文指南：https://docs.microsoft.com/zh-cn/windows/wsl/install-win10
+- 英文指南：https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+官方文档对 WSL 安装配置中可能遇到的常见问题都给出了详细的解决方案，请认真阅读。
+
+WSL 可以安装不同的 Linux 发行版，但目前官方并未提供 Fedora 发行版的安装。
+推荐使用 WSL 安装 Ubuntu 20.04 LTS 或 Ubuntu 20.10。
+
+.. note::
+
+   想在 WSL 上安装 Fedora 的读者可以参考以下安装指南：
+
+   - https://fedoramagazine.org/wsl-fedora-33/
+   - https://suiahae.me/Using-Fedora-33-on-Windows-10-WSL2/
+
+   需要注意的是，我们尚未验证以上安装指南是否有效。
+
+.. note::
+
+   开启 WSL 后，Linux 发行版的默认安装位置是 C 盘。安装多个发行版会占用 C 盘的大量空间。
+   可以先将一个新安装的 Linux 发行版导出备份，然后将其还原到其它盘，最后删除 C 盘上的发行版。
+   这样可以保证 C 盘有足够的空间。
+ 
```

Comment:
I feel it's difficult to understand the note without showing the related commands. I think it makes more sense moving it to the next section.
```suggestion
```